.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,61 @@
+name: Benchmarks
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**.py"
+      - "**.ipynb"
+      - "setup.cfg"
+      - ".github/workflows/**.yml"
+      - "requirements_dev.txt"
+      - "requirements.txt"
```

Comment:
Hmmm, its an error. When I was basing this new action in the pysyft benchmark and sympc tests I forgot to delete those. We want to delete them, otherwise, the benchmarks will be updated even when the changes are not being pushed to main.
If am I right, please, tell it to me and I will change it right now.

Thanks for the feedback!

Reply:
@aanurraj I have already done the change. Whenever you want, feel free to resolve the issue if I have done it correctly. Thanks!

Reply:
I made a comment below @danielorihuela